src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +607,26 @@
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+//		Disabled until it gets fixed (being sent at least 3 times on both Paper and Spigot)
+//		See https://github.com/PaperMC/Paper/issues/4844#issuecomment-1047067975
+//		if (Skript.classExists("org.bukkit.event.inventory.PrepareAnvilEvent")) {
+//			Skript.registerEvent("Anvil Prepare", SimpleEvent.class, PrepareAnvilEvent.class, "anvil prepar(e|ing)")
+//				.description("Called when an item is put in a slot for repair by an anvil")
+//				.requiredPlugins("Minecraft 1.9 or newer")
+//				.examples("on anvil prepare:",
+//					"\tevent-itemstack is set",
+//					"\tchance of 20%:",
+//					"\t\tadd anvil repair cost to player's level",
+//					"\t\tsend \"You're are LUCKY! You got your levels back.\" to player")
+//				.since("INSERT VERSION");
+//		}
+		if (Skript.classExists("com.destroystokyo.paper.event.block.AnvilDamagedEvent")) {
+			Skript.registerEvent("Anvil Damage", SimpleEvent.class, AnvilDamagedEvent.class, "anvil damag(e|ing)")
+				.description("Called when an anvil is damaged from being used")
```

Comment:
A) No. I haven't done much tests on which event is actually triggered when this case happens, it will be nice to have in another PR
B) Yes. later I might add `AnvilDamagedEvent#isBreaking` method